meetings/azure-solutions-architect/azure-solutions-architect-qa-2020-11-26.md

```diff
@@ -0,0 +1,51 @@
+# #azure-solutions-architect Wykorzystanie Azure AD w aplikacjach
```

Comment:
chyba moglibyśmy zmienić nazwę plików i wywalić qa z środka ;)

Reply:
To możemy załatwić globalnie oddzielnym PR. Chociaż nie jestem do końca przekonany :)

Reply:
spoko 😃